bazel/rules/library_rule.bzl

```diff
@@ -25,9 +25,11 @@ def stratum_cc_library(name, deps = None, srcs = None, data = None,
 
   alwayslink = 0
   if srcs:
-    if [s for s in srcs if not s.endswith(".h")]:
+    if type(srcs) == "select":
```

Comment:
This is needed for `select` sources. I looked through all the targets we build on CI, and all of them had `      alwayslink = 1` applied. So I think it's save to do in this case.